app/response_processor.py

```diff
@@ -1,17 +1,19 @@
-from app import receipt
 from app import settings
+from app.queue_publisher import QueuePublisher
 from app.settings import session
+from json import dumps
 from requests.packages.urllib3.exceptions import MaxRetryError
 
 
+def queue_receipt(logger, payload):
+    publisher = QueuePublisher(logger, settings.RABBIT_URLS, settings.RABBIT_RRM_RECEIPT_QUEUE)
```

Comment:
Do you need to instantiate every time you need to publish or can you reuse a publisher instance?